### PR TITLE
🐛 [Frontend] Fix Service browser

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Fix the Service Browser listing.
- [x] Fix the ``undefined`` unit in the Disk Usage's tooltip

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
:+1:

**Comment by matusdrobuliak66** [comment]
Required for https://github.com/ITISFoundation/osparc-simcore/issues/6441

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by GitHK — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/utils/Utils.js [hunk 1]
```diff
@@ -720,12 +720,14 @@ qx.Class.define("osparc.utils.Utils", {
...
+
       const k = 1000;
-      const sizes = ["Bytes", "KB", "MB", "GB", "TB"];
       const dm = decimals < 0 ? 0 : decimals;
-
-      const i = Math.floor(Math.log(bytes) / Math.log(k))
-      return `${isDecimalCollapsed ? parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) : (bytes / Math.pow(k, i)).toFixed(dm)} ${sizes[i]}`
+      const i = Math.floor(Math.log(bytes) / Math.log(k));
+      const value = (bytes / Math.pow(k, i)).toFixed(dm);
+      const metrics = ["Bytes", "kB", "MB", "GB", "TB"];
```
[the remark this answers is not in the document]
Comment (pcrespov) [reply services/static-webserver/client/source/class/osparc/utils/Utils.js:728]:
```javascript
function formatBytes(bytes, dm = 2, binary = false) {
    const k = binary ? 1024 : 1000;
    const sizes = binary
        ? ["Bytes", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"]
        : ["Bytes", "kB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
    if (bytes === 0) return '0 Bytes';
    
    const i = Math.floor(Math.log(bytes) / Math.log(k));
    const value = (bytes / Math.pow(k, i)).toFixed(dm);
    
    return `${value} ${sizes[i]}`;
}
```

[the remark this answers is not in the document]
Comment (odeimaiz) [reply services/static-webserver/client/source/class/osparc/utils/Utils.js:728]: Yes, I know. This has been discussed many, many, times in the last seven years... And we agreed that we won't change it again.

[the remark this answers is not in the document]
Comment (pcrespov) [reply services/static-webserver/client/source/class/osparc/utils/Utils.js:728]: old people needs reminders ;-)

**Review by sanderegg — COMMENTED** [review]
👍

**Pull request merged**